src/patternfly/upgrade-examples/PopoverUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,13 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
+| -- | -- | -- | -- | -- |
+| `.popover` | `<dvi>` | `.pf-c-popover__content` | `<div>` | Creates the container for the popover. |
+| `.pficon-close` | `<span>` |`.pf-c-popover__close` | `<div>` | Creates the closing button for the popover. |
```

Comment:
change this to compare `close` with `pf-c-popover__close`. pficon-close is the icon class not the button.

Reply:
class correction added to usage chart.